cdap-ui/app/cdap/components/Administration/AdminManagementTabContent/AdminManagementTabContentVariables.scss

```diff
@@ -0,0 +1,24 @@
+/*
+* Copyright © 2019 Cask Data, Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License"); you may not
+* use this file except in compliance with the License. You may obtain a copy of
+* the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+* WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+* License for the specific language governing permissions and limitations under
+* the License.
+*/
+
+$table_cell_border_color: gray;
+$table_background_color: transparent;
+$table_border_color: lightgray;
+$instance_bg_color: $grey-05;
+$instance_bg_hover_color: white;
+$height_of_header: 35px;
+$height_of_conent: 415px;
```

Comment:
typo for `content`.

Also is this intentional having it with underscore instead of dash? everywhere else seemed to be using dash

Reply:
Not sure when we added this during admin view development. Not too strong about the naming, will fix the typo and the usage of hyphens.